packages/static-hosting/lib/static-hosting.ts

```diff
@@ -221,226 +170,107 @@ export class StaticHosting extends Construct {
       : undefined;
 
     if (loggingBucket) {
-      loggingBucket.grantWrite(oai);
-
       new CfnOutput(this, "LoggingBucketName", {
         description: "CloudFront Logs",
         value: loggingBucket.bucketName,
         exportName: `${exportPrefix}LoggingBucketName`,
       });
     }
 
-    const loggingConfig = loggingBucket ? { bucket: loggingBucket } : undefined;
+    const s3Origin = new S3Origin(bucket);
+
+    const originRequestPolicy = new OriginRequestPolicy(
+      this,
+      "S3OriginRequestPolicy",
+      {
+        headerBehavior:
+          OriginRequestHeaderBehavior.allowList("x-forwarded-host"),
+      }
+    );
 
-    let originConfigs = new Array<SourceConfiguration>();
+    const originCachePolicy = new CachePolicy(this, "S3OriginCachePolicy", {
+      headerBehavior: CacheHeaderBehavior.allowList("x-forwarded-host"),
+      enableAcceptEncodingBrotli: true,
+      enableAcceptEncodingGzip: true,
+    });
 
-    // Add the backend host as an origin
```

Comment:
This is a feature I use all the time, why are we removing it?

Reply:
This has now been added back.